cli/installer/constants.go

```diff
@@ -0,0 +1,3 @@
+package installer
+
+const createIssueMsg = "If you need help, please create an issue: https://github.com/kubeshop/tracetest/issues/new/choose"
```

Comment:
this could go to the `tracetest.go`, `installer.go` or even `util.go` file and we can remove this useless file